Recreated the react-todo app by following tutorials

Daphineatim opened the pull request:
### In this project, the following were implemented;

- Created a new repository and used git-flow
- Initialized react by running the create-react-app command
- Created react components to divide the UI
- Used functional components
- Used react hooks to manage the state and data of the UI

Daphineatim committed:
Recreated the react-todo app by following tutorials

Daphineatim committed:
Fix linters

Daphineatim committed:
Fix linters

Daphineatim committed:
Fix linters

Daphineatim committed:
Fix linters

Daphineatim committed:
Fix linters

Daphineatim committed:
Fix linters

Daphineatim committed:
Update README.md

gandradep commented:
Hi @Daphineatim 
![360_F_399825302_hyLIim743GeXzW6snSCaMtYVbPu6a438](https://user-images.githubusercontent.com/8645872/199662650-f5596eae-b8aa-4189-b910-4a4be55609ad.jpg)

Great job !!! You are almost there!!

To Highlight! 👏 🟢
No linter errors ✔️
Descriptive pull request ✔️
Descriptive commit messages ✔️
Descriptive README ✔️

Suggestion
- In your modules src folder, you have a component folder that is not imported anywhere else in your code. It is good practice to leave any unused pieces of code.
- To follow the gitFlow rules, the project should have a dev branch instead of the main. Please create and set it up as default branch.
- In the 'TodoContainer.js' file there are commented methods that could be erased.

Optional suggestions
- The project could have more commits that show progress. For example, you can commit after creating every component.

N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Daphineatim committed:
Delete commented code in the TodoContainer.js file

Daphineatim commented:
Thank you for the feedback, I have changed the default branch from main to development and also deleted the commented code in the TodoContainer.js file.

gandradep commented:
@Daphineatim Great job !!!, now it is time to merge.
